react/src/reactTextMask.js

```diff
@@ -52,12 +37,12 @@ export const MaskedInput = React.createClass({
     return (
       <input
         {...props}
-        onInput={this.onChange}
+        onInput={event => this.onChange(event)}
         defaultValue={this.props.value}
-        ref={(inputElement) => (this.inputElement = inputElement)}
+        ref={inputElement => this.inputElement = inputElement} //eslint-disable-line
```

Comment:
<img width="681" alt="screen shot 2017-04-14 at 12 34 04 pm" src="https://cloud.githubusercontent.com/assets/449136/25049219/ac739a8c-210e-11e7-9265-5296716e14c0.png">
 Without that line I get this.